programs/survival/graves.sc

```diff
@@ -0,0 +1,349 @@
+// Tweak Stuff
+    global_request_time = 60;   //Time between 2 requests, if requests is turned on (in seconds)
+    global_allow_request = true;   //If enabled, does not send unlocking request to player
+    global_allow_only_owner = false; //Only grave owners are allowed to open their graves
+    global_operator_permission_level = 2; //Operator permission level
+
+
+//Boring Stuff*****************************************************************************************************************************************
+__config()->{
+    'stay_loaded' -> true,
+    'scope' -> 'global',
+    'requires' -> {
+        'carpet' -> '>=1.4.57'
+    }
+};
+
+__command() -> null;
+
+gdata_file = read_file('gdata','JSON');
```

Comment:
Wouldn't it be easier to use `store_app_data()` and do it in NBT? unless you want JSON specifically?

Reply:
When I first started learning about data storage concepts, my reference was waypoints app. Since it has to store multiple files, ig firi used this method. I sorta never stopped using this after learning from it. If there is a difference in performance, Imma go ahead and change it.

Reply:
store_app_data is the most performance-effective way, as well as taking up less space to store the info